sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -89,19 +87,21 @@
  *   .apply(...)
  *   .apply(JdbcIO.write()
  *      .withDataSource(myDataSource)
- *      .withQuery(query)
- *      .withPreparedStatementSetter(new JdbcIO.PreparedStatementSetter<MyElement>() {
+ *      .withStatement("insert into Person values(?, ?)")
+ *      .withPreparedStatementSetter(new JdbcIO.PreparedStatementSetter<KV<Integer, String>>() {
```

Comment:
This line uses KV[Integer,String] but the next line uses MyElement - please fix.